hydrolib/core/io/ini/parser.py

```diff
@@ -289,3 +290,13 @@ def _is_datarow(self, _: str) -> bool:
         # we assume that we already checked whether it is either a comment,
         # section header or a property
         return self._config.parse_datablocks
+
+    @classmethod
+    def parse(cls, filepath: Path):
+        parser = cls(ParserConfig())
```

Comment:
would you not want to pass the `ParserConfig` with a default parameter? 

```suggestion
    def parse(cls, filepath: Path, config=None):
        if config is None:
            config = ParserConfig()
        parser = cls(config)
```